src/helpers/scaffoldProject.ts

```diff
@@ -26,30 +27,64 @@ export const scaffoldProject = async ({
   const spinner = ora(`Scaffolding in: ${projectDir}...\n`).start();
 
   if (fs.existsSync(projectDir)) {
-    if (fs.readdirSync(projectDir).length === 0) {
+    const userFilesSet = new Set(getAllFiles(projectDir));
+    const srcFiles = [...getAllFiles(srcDir), ".gitignore"]; // template files
```

Comment:
this means we are just checking against the base app without any addons. If there is files that would conflict with any of these other ones these would not be matched and potentially removed by accident